NuRadioMC/utilities/Veff.py

```diff
@@ -125,8 +130,7 @@ def get_Aeff_proposal(folder, trigger_combinations={}, station=101):
         out['energy'] = E
 
         weights = np.array(fin['weights'])
-        # triggered = np.array(fin['triggered'])
-        triggered = get_triggered(fin)
```

Comment:
shouldn't we keep the call to the `get_triggered` function here?

Reply:
That 'triggered' is only used to check if there are triggering events in the file, so it's faster to take the fin['triggered'] array, and the effect is the same.